editors/code/package.json

```diff
@@ -1349,4 +1358,4 @@
             ]
         }
     }
-}
+}
```

Comment:
nit: missing trailing newline

Reply:
Thank you so much - just realised there's a vscode extension 'Formatting Toggle'. Am always reaching into the settings to flick it on and off - now I don't have to. Result!

Reply:
You can also use ctrl+shift+i to format the current file once.